jsapp/js/components/formLanding.es6

```diff
@@ -129,8 +134,31 @@ export class FormLanding extends React.Component {
       asset: this.state
     });
   }
+  increaseDVCOUNT_LIMIT() {
+    if (this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM <= this.state.deployed_versions.count + DVCOUNT_LIMIT_MINIMUM) {
+      this.setState({ DVCOUNT_LIMIT : this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM });
+    }
+    if (this.state.DVCOUNT_LIMIT >= 80) {
+      var curr = this.state.currentVersions;
+      if (this.state.nextPageUrl) {
+        dataInterface.loadNextPageUrl(this.state.nextPageUrl).done((data) => {
+          this.setState({nextPageUrl: data.deployed_versions.next});
+          Object.values(data.deployed_versions.results).forEach(function(ele){
+              curr.push(ele);
+          });
+        });
+      }
+      this.setState({currentVersions: curr});
+    }
+  }
   renderHistory () {
     var dvcount = this.state.deployed_versions.count;
+    if (this.state.nextPageUrl === 'inital') {
+      this.setState({
+          nextPageUrl: this.state.deployed_versions.next,
+      });
+      this.state.currentVersions = this.state.deployed_versions.results;
```

Comment:
I added it because I wanted a way to reference the return of loading the next page, and I could not reference `this` in the response to my `dataInterface` call. Although your code below solved this issue I'm still not entirely sure why. It appears like I'm not doing anything to the `curr` var anymore yet the code needs `curr.push(ele)` to work...